setup.py

```diff
@@ -49,9 +49,8 @@ def readme():
         'Programming Language :: Python',
         'Topic :: Scientific/Engineering',
     ],
-    url              = 'https://github.com/chemos-inc/phoenics',
-    author           = 'ChemOS Inc.',
-    author_email     = 'florian@chemos.io',
+    url              = 'https://github.com/Atinary-technologies/phoenics',
+    author           = 'Atinary Technologies',
```

Comment:
author should be:
`"Atinary Technologies Inc. & Atinary Technologies Sarl"`

Reply:
done